Converter/Calculator/Converter.cs

```diff
@@ -0,0 +1,140 @@
+
+using System.Diagnostics;
+
+namespace Calculator;
+
+public static class Converter
+{
+    public static void ConvertTo(int value)
```

Comment:
Your converter must Convert values. It means that it receives everything it needs for conversion and returns results then.